sites/partners/src/listings/PaperListingForm/sections/ApplicationDates.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import { useFormContext } from "react-hook-form"
+import { t, GridSection, DateField, TimeField } from "@bloom-housing/ui-components"
+import { FormListing } from "../index"
+import moment from "moment"
```

Comment:
Do we still want to use `moment`? Maybe it's a good moment to add `dayjs` and then replace other moment instances in a separated PR?

Reply:
I think a change like that should probably be handled in a separate PR so it stays consistent if that's ok